gnmi_server/server_test.go

```diff
@@ -402,46 +402,64 @@ func runServer(t *testing.T, s *Server) {
 }
 
 func getRedisClientN(t *testing.T, n int, namespace string) *redis.Client {
+	addr, err := sdcfg.GetDbTcpAddr("COUNTERS_DB", namespace)
+	if err != nil {
+		t.Fatalf("failed to get addr %v", err)
+	}
 	rclient := redis.NewClient(&redis.Options{
 		Network:     "tcp",
-		Addr:        sdcfg.GetDbTcpAddr("COUNTERS_DB", namespace),
+		Addr:        addr,
 		Password:    "", // no password set
 		DB:          n,
 		DialTimeout: 0,
 	})
-	_, err := rclient.Ping().Result()
+	_, err = rclient.Ping().Result()
 	if err != nil {
 		t.Fatalf("failed to connect to redis server %v", err)
 	}
 	return rclient
 }
 
 func getRedisClient(t *testing.T, namespace string) *redis.Client {
-
+	addr, err := sdcfg.GetDbTcpAddr("COUNTERS_DB", namespace)
+	if err != nil {
+		t.Fatalf("failed to get addr %v", err)
+	}
+	db, err := sdcfg.GetDbId("COUNTERS_DB", namespace)
```

Comment:
>:= [](http://example.com/codeflow?start=9&length=2)

Do you want to use `=` instead of `:=` ?

Reply:
":=" is used for declaration of "db".
If I use "=" instead, I need to declare "db" at first: 
var db int
db, err := sdcfg.GetDbId("COUNTERS_DB", namespace)